ocaml/xapi/xapi_globs.ml

```diff
@@ -1470,6 +1473,11 @@ let other_options =
     , (fun () -> string_of_int !max_observer_file_size)
     , "The maximum size of log files for saving spans"
     )
+  ; ( "secureboot-enforce"
+    , Arg.Set secureboot_enforce
+    , (fun () -> string_of_bool !secureboot_enforce)
+    , "Do not start a VM with no SB certificates if secureboot is set to on"
```

Comment:
I think using "block" creates the expectations that the system will return an error in this situation
```suggestion
    , "Block secureboot VMs from starting when the secureboot certificates are not valid"
```

Reply:
`Block a secureboot VM from starting when its secureboot certificates are not valid` - the singular is usually easier to understand; we are obviously talking about the general case and it would apply to all such VMs implicitly.